src/re_frame/loggers.cljc

```diff
@@ -0,0 +1,36 @@
+(ns re-frame.loggers
+  (:require
+    [clojure.set :refer [difference]]))
+
+;; "loggers" holds the current set of logging functions.
+;; By default, re-frame uses the functions provided by js/console.
+;; Use set-loggers! to change these defaults.
+
+(def ^:private loggers
+  (atom {:log       #?(:cljs (js/console.log.bind   js/console)
+                       :clj  println)
+         :warn      #?(:cljs (js/console.warn.bind  js/console)
+                       :clj  (partial println "WARN: "))
```

Comment:
We could.  I didn't on the assumption that in tests you probably don't care, and I didn't want to introduce a new dependency -- even if it is as harmless as `tools.logging`.  (Especially given that re-frame has obviously chosen not to use a ClojureScript logging framework even in prod code, presumably for a similar reason.)  Is there a benefit to using tools logging in tests rather than just `println`?